lib/miyuki/notifier.rb

```diff
@@ -19,7 +19,12 @@ def initialize(notifier = nil)
     end
 
     def notify(title, message)
-      @notifier.notify(title, message) if has_notifier?
+      begin
+        @notifier.notify(title, message) if has_notifier?
+      rescue NoMethodError
+        #libnotify is not installed, maybe.
+        #someone may not want to install gnome on its server :V
```

Comment:
May I remove notifications from test' configurations? If we need to test them, we need them turned on, and `rake test` will die if libnotify (on servers, for example) is not installed.